### ISSUE TITLE
Show average component load time some how

### ISSUE THREAD

**Issue by kenchris** [webcomponents/webcomponents.org#784]
Developers should be very concerned about load time today, so it would be nice to show the current time it took to load and show the component. If some average can be shown instead that would be even better, ie. average time on common devices or similar.

@slighlylate

**Comment by samuelli** [comment]
Yep, sounds good! The challenging part is ensuring they are accurate & representative numbers. eg. do we measure an empty element? Do we measure all the elements/some of them? Load time is one factor, stamping is another. For a checkbox, the requirements are pretty different to a singleton app level element.

**Comment by seaneking** [comment]
Cool idea, but I'd echo @samuelli's concern that a single 'load time' metric would be pretty hard to get, and very hard to correlate to actual perceived perf. Is it just time taken to load/parse/upgrade the element? Is it first paint? What about components that rely on network requests/APIs? etc.

Still, probably better than nothing. Just might encourage people to optimize for the wrong things.

**Comment by seaneking** [comment]
Could also show element size (gzipped) in the meta info on the left somewhere

**Comment by samuelli** [comment]
@seaneking: including or excluding deps? The whole repo or just the specified files in bower 'main' (+ loaded graphs)

**Comment by seaneking** [comment]
Hmm yeah that's tricky, people might lazy-load deps so bundled size isn't really an accurate figure for load perf either
